eventsourcing/src/test/java/org/axonframework/eventsourcing/eventstore/AggregateBasedStorageEngineTestSuite.java

```diff
@@ -204,15 +204,18 @@ void eventsWithoutTagsAreNotSourcedAsAggregatedEvents() {
     }
 
     @Test
-    void tagsNotMatchingCriteriaAreRejected() {
-        var result = testSubject.appendEvents(AppendCondition.withCriteria(TEST_AGGREGATE_CRITERIA),
-                                              taggedEventMessage("event-4", OTHER_AGGREGATE_CRITERIA .tags()),
-                                              taggedEventMessage("event-5", OTHER_AGGREGATE_CRITERIA.tags()),
-                                              taggedEventMessage("event-6", OTHER_AGGREGATE_CRITERIA.tags()))
-                                .thenCompose(AppendTransaction::commit);
-
-        var actualException = assertThrows(ExecutionException.class, () -> result.get(1, TimeUnit.SECONDS));
-        assertInstanceOf(IllegalArgumentException.class, actualException.getCause());
+    void eventsWithTagsNotMatchingCriteriaAreInsertedAtSequenceZero() {
```

Comment:
Aaaah, and here's the test for my comment in the Legacy store. Assumption confirmed (right?!).

Reply:
Yes, indeed.